### ISSUE TITLE
[SUGESTÃO]

### ISSUE THREAD

**Issue by ericasilverio2210** [elidianaandrade/dio-lab-open-source#6951]
**Sua sugestão está relacionada a um problema? Por favor descreva.**
Não

**Descreva a solução que você gostaria**
Senti muita dificuldade para criar meu primeiro repositório e enviar um projeto. Foi através de vídeos no YouTube que consegui. Achei o material um pouco confuso, somente depois de ter pesquisado outras fontes, foi que consegui entender o que estava sendo proposto.

**Descreva as alternativas que você considerou**
Acredito que as vídeo - aulas poderiam ser mais objetivas, com descrições mais detalhadas da finalidade de determinada ação que está sendo demonstrada. Para quem está tendo os primeiros contatos com Git GitHub, acredito que seja difícil assimilar todo o material.

**Contexto adicional**
N/A

**Comment by SamSantos7** [comment]
Concordo, tem termos falados nos videos que não entendo e parece que os videos são para pessoas que já trabalham nessa área e vão entender o que é dito com clareza. Poderia ser mais simplificado o modo de explicação.
